scripts/check_license.sh

```diff
@@ -87,5 +86,14 @@ if [ "$(<README.md grep "${READMECOPYRIGHT}" | wc -l | tr -d ' ')" = "0" ]; then
     RETURN_VALUE=1
     echo "README.md file need to have it's license date range updated."
 fi
+
+if [ $RETURN_VALUE -ne 0 ] && [ $VERBOSE == "true" ] ; then
+    echo -e "\n${RED_FG}FAILED LICENSE CHECK.${END_FG_COLOR}"
+    if [ $INPLACE == "false" ]; then
```

Comment:
In the case where we already modified the files, you might to write something different, like "5 files were modified" or something like that.